Move dependencies to be in npm

Is there any change to move all dependencies to npm?
https://github.com/jsdoc3/jsdoc/blob/master/package.json#L25

I'm behind a corporate firewall where npm packages have to be requested and then cloned to an internal private npm repo, and breaking out of npm packages means npm goes direct to github.

I'm definitely getting ENOGIT though...

I'm getting this as a transitive dependency of grunt-jsdoc, which requires version 3.2.2. Would upgrading help? 

```
error	23-Mar-2015 08:53:59	npm ERR! git clone https://github.com/hegemonic/taffydb.git undefined
error	23-Mar-2015 08:53:59	npm ERR! git clone https://github.com/hegemonic/taffydb.git undefined
error	23-Mar-2015 08:54:08	npm ERR! Linux 2.6.32-220.23.1.el6.x86_64
error	23-Mar-2015 08:54:08	npm ERR! argv "node" "/usr/bin/npm" "install"
error	23-Mar-2015 08:54:08	npm ERR! node v0.10.33
error	23-Mar-2015 08:54:08	npm ERR! npm  v2.7.0
error	23-Mar-2015 08:54:08	npm ERR! code ENOGIT
error	23-Mar-2015 08:54:08	
error	23-Mar-2015 08:54:08	npm ERR! not found: git
```

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "marked": "~0.3.6",
     "requizzle": "~0.2.1",
     "strip-json-comments": "~2.0.1",
-    "taffydb": "https://github.com/hegemonic/taffydb/tarball/7d100bcee0e997ee4977e273cdce60bd8933050e",
+    "taffydb": "2.6.2",
     "underscore": "~1.8.3"
   },
   "devDependencies": {

